src/components/Keyword/KeywordFilter/styles.js

```diff
@@ -1,175 +1,174 @@
 `;
 
 export const ItemUnderBar = styled.div`
-    width:32px;
-    height:8px;
-    background-color:#222222;
-    position:absolute;
-    top:216px;
-    left:${props => {
-            if(props.toggle){
-                if(props.menu === '전체'){
-                    return '488px';
-                }else if(props.menu === '아우누리'){
-                    return '568px';
-                }else if(props.menu === '아우미르'){
-                    return '670px';
-                }else{
-                    return '830px';
-                }
-                
-            }else{
-                if(props.menu === '전체'){
-                    return '353px';
-                }else if(props.menu === '아우누리'){
-                    return '435px';
-                }else if(props.menu === '아우미르'){
-                    return '533px';
-                }else{
-                    return '695px';
-            }
-        }
-    }}
+  width: 32px;
+  height: 8px;
+  background-color: #222222;
+  position: absolute;
+  top: 216px;
+  left: ${(props) => {
+    if (props.toggle) {
+      if (props.menu === '전체') {
+        return '488px';
+      } else if (props.menu === '아우누리') {
+        return '568px';
+      } else if (props.menu === '아우미르') {
+        return '670px';
+      } else {
+        return '830px';
+      }
+    } else {
+      if (props.menu === '전체') {
+        return '353px';
+      } else if (props.menu === '아우누리') {
+        return '435px';
+      } else if (props.menu === '아우미르') {
+        return '533px';
+      } else {
+        return '695px';
+      }
+    }
+  }};
 `;
 
 export const FilterList = styled.nav`
-    min-width:1310px;
-    height:36px;
-    display:flex;
-    align-items:center;
-    font-size:12px;
-    position:absolute;
-    left:${props=>props.toggle?'501px':'366px'};
-    top:263px;
-
-    .read,
-    .notread {
-        margin-right:15px;
-    }
-
-    .goStore {
-        margin-right:10px;
-        display:flex;
-        align-items:center;
-    }
-
-    .delete {
-        margin-right:24px;
-        display:flex;
-        align-items:center;
-    }
+  min-width: 1310px;
+  height: 36px;
+  display: flex;
+  align-items: center;
+  font-size: 12px;
+  position: absolute;
+  left: ${(props) => (props.toggle ? '501px' : '366px')};
+  top: 263px;
+
+  .read,
+  .notread {
+    margin-right: 15px;
+  }
+
+  .goStore {
+    margin-right: 10px;
+    display: flex;
+    align-items: center;
+  }
+
+  .delete {
+    margin-right: 24px;
+    display: flex;
+    align-items: center;
+  }
 `;
 
 export const CheckBox = styled.div`
-    width:16px;
-    height:16px;
-    margin-right:24px;
-    border-radius:3px;
-    border:1px solid #c4c4c4;
-    cursor:pointer;
-
-    ${props => {
-        if(props.checkAll){
-            return `
+  width: 16px;
+  height: 16px;
+  margin-right: 24px;
+  border-radius: 3px;
+  border: 1px solid #c4c4c4;
+  cursor: pointer;
+
+  ${(props) => {
+    if (props.checkAll) {
+      return `
```

Comment:
분기문으로 나눠지는게 아니라 그냥 `&&` 써도 되겠네요

```suggestion
  ${(props) => props.checkAll && `/* ... */ `
```

Reply:
아 &&써도 가능하네요 감사합니다!

Reply:
3be1689